flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalog.java

```diff
@@ -480,10 +478,7 @@ public void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
             throw new DatabaseNotExistException(getName(), tablePath.getDatabaseName());
         }
 
-        boolean managedTable = ManagedTableListener.isManagedTable(this, resolvedTable);
-        Table hiveTable =
-                HiveTableUtil.instantiateHiveTable(
-                        tablePath, resolvedTable, hiveConf, managedTable);
+        Table hiveTable = HiveTableUtil.instantiateHiveTable(tablePath, resolvedTable, hiveConf);
```

Comment:
We should sync the related changes to hive connector repo since it has been externalized.